src/main/java/tests/OperationHandlersTest.java

```diff
@@ -0,0 +1,65 @@
+package tests;
+
+import core.basesyntax.*;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class OperationHandlersTest {
+    @Test
+    public void testBalanceOperation() {
+        OperationHandler handler = new BalanceOperation();
+        Map<String, Integer> storage = new HashMap<>();
+        FruitTransaction transaction = new FruitTransaction();
+        transaction.setOperation(Operation.BALANCE);
+        transaction.setFruit("banana");
+        transaction.setQuantity(100);
+        ((BalanceOperation) handler).handle(transaction, storage);
+
+        assertEquals(100, (int) storage.get("banana"));
```

Comment:
It is good practice to include a message in the `assertEquals` method to specify what the test is checking for. This makes it easier to understand the purpose of the test when it fails.